js/packages/web/src/components/Layout/index.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { Layout } from 'antd';
 
 import { AppBar } from '../AppBar';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Left this here because it looks like someone commented out the `Footer` in this file on purpose, perhaps to bring it back soon.